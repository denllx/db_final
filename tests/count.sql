CREATE DATABASE OOP;
USE OOP;
CREATE TABLE oop_info(stu_id INT NOT NULL, PRIMARY KEY(stu_id), stu_name CHAR);
INSERT INTO oop_info(stu_id, stu_name) VALUES (2018011243, "a");
INSERT INTO oop_info(stu_id, stu_name) VALUES (2018011344, "b");
INSERT INTO oop_info(stu_id, stu_name) VALUES (2018011445, "c");
INSERT INTO oop_info(stu_id, stu_name) VALUES (2018012345, "c");
INSERT INTO oop_info(stu_id, stu_name) VALUES (2018010143, "b");
INSERT INTO oop_info(stu_id) VALUES (2018019143);
SELECT COUNT(*) from oop_info;
SELECT COUNT(*) from oop_info where stu_id = 2018011243;
SELECT COUNT(stu_name) from oop_info where stu_name="c";
SELECT COUNT(stu_id) from oop_info where stu_name="b";
SELECT stu_id,stu_name,COUNT(stu_name) from oop_info where stu_id > 2016030040;
DROP DATABASE OOP;
