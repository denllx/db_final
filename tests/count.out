COUNT(*)
6
COUNT(*)
1
COUNT(stu_name)
2
COUNT(stu_id)
2
stu_id	stu_name	
2018010143	b	
2018011243	a	
2018011344	b	
2018011445	c	
2018012345	c	
2018019143	NULL	
COUNT(stu_name)
5
